public/locales/en/common.json

```diff
@@ -190,12 +191,12 @@
     "RENAME": "Rename",
     "CENTER_RENAMED": "Center Renamed Successfully!",
     "SEND_REQUEST": "Send Request",
-    "SURE_DELETE_CENTER": "Are you sure you want to Delete Center?",
+    "SURE_DELETE_CENTER": "Are you sure you want to Delete ",
     "CENTER_UPDATE_SUCCESSFULLY": "Center has been Successfully Updated",
     "CENTER_DELETE_SUCCESSFULLY": "Center has been Successfully Deleted",
     "CENTER_CREATED_SUCCESSFULLY": "Center Created Successfully",
     "SEARCHBAR_PLACEHOLDER": "Search Center..",
-    "NOT_ABLE_CREATE_CENTER": "*Not able to create center for selected Data"
+    "CENTER_UPDATE_FAILED": "Something went wrong to update data"
```

Comment:
@upendraTekdi change it to Something went wrong while updating center

Reply:
done